ls1intum/Artemis
# `Plagiarism`: Do not take practice participations into account

### Checklist
#### General


#### Server

### Motivation and Context


#5882

### Description

Test runs and therefore practice participations are no longer taken into consideration for plagiarism cases.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Students
- 1 Programming Exercise

1. Log in to Artemis
2. Navigate to Course Administration
3. Participate with both students before the deadline and afterwards using the practice mode and submit the same code
4. Check for plagiarism
5. Make sure that only the graded participations are taken into account

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage
